contracts/Fund.sol

```diff
@@ -503,6 +507,11 @@ contract Fund is ERC20, DestructiblePausable {
     return true;
   }
 
+  // Update the address of the data feed contract
+  function setDataFeed(address _address) onlyOwner {
```

Comment:
`require(_address != address(0));`

Reply:
decide if address error checking or return bool is needed here (and other address setting functions below) to facilitate testing